projects/addon-doc/components/api/api-item.style.less

```diff
@@ -54,6 +54,11 @@
 .t-input {
     min-inline-size: 10rem;
     margin-block-start: -0.625rem;
+
+    code {
+        overflow: hidden;
+        white-space: nowrap;
+    }
```

Comment:
Maybe more preferably add directly to https://github.com/taiga-family/taiga-ui/blob/main/projects/addon-doc/components/main/main.style.less#L44-L75

Reply:
Let's leave it here for now, and if we meet it again – we'll move it.